### DISCUSSION TITLE
Adding Slack Monitoring to this discussion topic.

### DISCUSSION THREAD

**Discussion by gilesknap** [epics-containers/epics-containers.github.io#115] [General]
I'm experimenting with using discussions in epics-containers docs repo as a primary communication medium backed by Slack.

If you want to join in you can simply post here via the GitHub UI.

If you want to get notifications then you can add the GitHub plugin to your Slack Workspace using this link https://slack.github.com/

Then issue these commands into slack which subscribe to discussions in this repo and then unsubscribe from the less useful notifications that came with the first command::
```
/github subscribe epics-containers/epics-containers.github.io discussions:{category:General}
/github unsubscribe epics-containers/epics-containers.github.io releases deployments commits
```

**Comment by gilesknap** [comment]
@ulrikpedersen please can you see if this works for you? I'm having some issues with seeing replies at the moment but might just be config.

**Reply by gilesknap** [reply]
If it works,  i suggest extending it to all topics in this discussion (its been quite quiet for the last few years)

**Reply by ulrikpedersen** [reply]
sorry, I didn't notice this earlier. I don't monitor github notifications closely - which is why slack integration would be great 👍 

Give me 10 min to check if I can add this to my slack and make it work...

**Reply by gilesknap** [reply]
I'm getting the impression that the integration currently only notifies on new topics and its one way only. That's a bit of a shame if I am right, but I figure they will fix it eventually. It definitely does threads for PRs.

**Comment by ulrikpedersen** [comment]
OK, I have subscribed:

> ✅  Subscribed to [epics-containers/epics-containers.github.io](https://github.com/epics-containers/epics-containers.github.io). This channel will receive notifications for
> `issues, pulls, commits, releases, deployments, discussions:'{"category":["General"]}'`

**Comment by ulrikpedersen** [comment]
I just got a notification (that I didn't particularly need) on this subscription:

>Deployment to [github-pages](https://github.com/epics-containers/epics-containers.github.io/actions/runs/8421914020/job/23059946186) by [github-pages[bot]](https://github.com/apps/github-pages)
Status
Completed
Commit
[0dd2601 (gh-pages)](https://github.com/epics-containers/epics-containers.github.io/commit/0dd26014aa8a328498d5e6621a42dc09e6a5c7e1)
Workflow
[pages build and deployment #237 / deploy](https://github.com/epics-containers/epics-containers.github.io/actions/runs/8421914020/job/23059946186)
<https://github.com/[epics-containers/epics-containers.github.io](https://github.com/epics-containers/epics-containers.github.io)|epics-containers/epics-containers.github.io>epics-containers/epics-containers.github.io | Today at 14:40

**Reply by gilesknap** [reply]
Yeah that happened to me to.  But luckily this is the docs repo and its not too noisy

**Reply by gilesknap** [reply]
(But  i'm guessing you did not get these replies as notifications?)

**Reply by ulrikpedersen** [reply]
yeah, didn't get these replies. Maybe only new topics give updates?

**Reply by ulrikpedersen** [reply]
I think I'll tweak my subscription a little to include all of the discussion categories (instead of just General) and remove the notifications on releases and deployments

**Reply by gilesknap** [reply]
I'm thinking it is new topics only which is a bit rubbish - I was hoping for threads in slack synced with topics and their replies

**Reply by ulrikpedersen** [reply]
Yup, the docs for the integration plugin seems to say that:

> discussions - Discussions created or answered

BTW I ended up configuring the plugin like this: `/github subscribe epics-containers/epics-containers.github.io discussions` and then take away the subscriptions I didn't want: `/github unsubscribe epics-containers/epics-containers.github.io releases deployments`

**Reply by ulrikpedersen** [reply]
Indeed when I marked our other discussion with an "answer" then I got a notification.

**Reply by gilesknap** [reply]
Thanks I'll try those settings too
